content/get-started/create-components/content.md

```diff
@@ -1,81 +1,168 @@
+
+
+**Is it possible to meet this user need by customising an existing component?**
+
+We could probably customise the `callout` component to meet this user need. However, it feels a bit confusing, cards are used for grouping and callouts are used for grouping text and highlighting it to a user. In this situation we will create the component but let the [community](https://community.digital.gov.au/) know that `card` and `callout` are very similar but meet different user needs.
+
+
+**Is it possible to use components that exist to make this?**
+
+For now we will be keeping it simple and not using any components. However, we will use the functionality from `core` and in the future we can add the `headings` for the card titles as a dependency.
+s
+
+**Can we build this component in a way that other products can benefit from it in the future?**
+
+Yes, we will be focusing on keeping this simple so we can share it with others.
+
+
+**Is there research or discussion about similar patterns on the community**
+
+Yes, there is a great [discussion on cards](https://community.digital.gov.au/t/cards/65/14). We will use this as a basis for the work that is being done.
+
+
+**What functions and variables in core can I use in this component?**
+
+We can use the functions to keep our components spacing consistent:
+- `AU-space` we can use for our padding inside the card
+- `AU-fontgrid` we can use for our typography inside the card
+
+We can use the following variables to keep the components appearance consistent:
+- `$AU-color`... variables for the colors
+- `$AU-border-radius` for the round corners
+
+We will need to create a variable for the shadow around the card as this currently doesn't exist.
+
+
+## Building a custom component
+
+Before you start creating components it is recommeneded to understand how to use BEM ( Block Element Modifier )[http://getbem.com/]. BEM allows our components styling to stay specific and customisable.
+
+To start we will create a simple HTML snippet for a card. The class `.card` is unique to this component and can be reused throughout the product. The HTML created doesn't use any classes from the design system, it may be recommended to wrap this component in `au-body` or maybe this component can work without it.
+
 ```html
-<div class="card"></div>
+<div class="card">
+  <p>This is our card</p>
+</div>
 ```
 
+We now create our styles for the card. It is important to understand the [colour system](/get-started/customise-color) and the [spacing functions](/get-started/font-size-space) to get the most benefits out of the system. The [core component](/components/core/code) comes with a lot of functions and variables to make your life easier. Core won't have everything you need and you will need to create your own variables e.g.`$boxshadow`.
+
```

Comment:
Should we mention the below is SASS and not CSS?

Reply:
I have changed how these parts are presented.